src/main.cpp

```diff
@@ -194,7 +197,23 @@ int main(int argc, char *argv[])
 				"and puts the standard output of that process into a dialog window in Fritzing.\n"
 				"The process path follows the -ep argument; the name of the menu item follows the -epname argument;\n"
 				"and any arguments to pass to the external process are provided in the -eparg arguments.\n";
+            break;
 		}
+        case FInitResultVersion:
+        {
+            QTextStream cout(stdout);
+            cout <<
+                "Fritzing " << Version::versionString() << "\n"
+                "Copyright (C) 2016 ???.\n"
+                "Source code License GPLv3: GNU GPL version 3 <https://www.gnu.org/licenses/gpl.html>.\n"
+                "Documentation and Part Designs License CC BY-SA 3.0: Creative Commons Attribution-ShareAlike 3.0 Unported <https://creativecommons.org/licenses/by-sa/3.0/legalcode>.\n"
+                "This is free software: you are free to change and redistribute it.\n"
+                "There is NO WARRANTY, to the extent permitted by law.\n"
+                "\n"
+                "Maintained by the Friends-of-Fritzing e.V.\n";
```

Comment:
@KjellMorgenstern is this right?

Reply:
After some discussion with @KjellMorgenstern, we've think it's best for now to not have the © in the `--version` message due to the complicated legal situation of Fritzing right now.
In order not to hold this PR back: can you strip this message to just the version print? We'll revisit the © part later.